README.md

```diff
@@ -42,5 +42,66 @@ Client Authentication: Send as Basic Auth Header
 - FreeAgent App Signup for Sandbox:
   - https://signup.sandbox.freeagent.com
 
+### Refreshing the Access Token Automatically
+
+You can set up a [pre-request script](https://learning.postman.com/docs/postman/scripts/pre-request-scripts/) if you'd like to automate the process of refreshing your access token once it expires. To to this:
+
+- In Postman, click on the icon with three dots next to the name of the imported FreeAgent collection.
+- Select **Edit**
+- Open the **Pre-request Scripts** tab and paste in the following:
+
+```
+// Refresh the Access Token if it expired
+let tokenDate = new Date(2020, 5, 14);
+let tokenTimestamp = pm.environment.get("oauth_timestamp");
+if (tokenTimestamp) {
+  tokenDate = Date.parse(tokenTimestamp);
+}
+let expiresInTime = pm.environment.get("expires_in");
+if (!expiresInTime) {
+  expiresInTime = 300000; // Set default expiration time to 5 minutes
+}
+
+if ((new Date() - tokenDate) >= expiresInTime) {
+  const refreshAccessTokenRequest = {
+    url: pm.environment.get("token_url"),
+    method: 'POST',
+    header: 'Accept:application/json',
+    body: {
+      mode: 'urlencoded',
+      urlencoded: [
+        { key: "client_id", value: pm.environment.get("client_id"), disabled: false },
+        { key: "client_secret", value: pm.environment.get("client_secret"), disabled: false },
+        { key: "refresh_token", value: pm.environment.get("refresh_token"), disabled: false },
+        { key: "grant_type", value: 'refresh_token', disabled: false }
+      ]
+    }
+  };
+  pm.sendRequest(refreshAccessTokenRequest, function (err, res) {
+    if (err) {
+      console.log("Ensure you have the client_id, client_secret, token_url and refresh_token set as environment variables.")
```

Comment:
```suggestion
      console.error("Ensure you have the client_id, client_secret, token_url and refresh_token set as environment variables.");
      return;
```
Given this will be an error case. Related, if we hit this block and report an error, it's probably better to return after logging.

Reply:
Updated this as well.